apps/gateway/lib/gateway/controllers/curse_of_mirra/quest_controller.ex

```diff
@@ -44,4 +46,19 @@ defmodule Gateway.Controllers.CurseOfMirra.QuestController do
         |> send_resp(400, "Error completing bounty")
     end
   end
+
+  def add_quests(conn, %{"user_id" => user_id, "type" => type, "amount" => amount}) do
+    with {:ok, _changes} <- Quests.add_quests_to_user_id_by_type(user_id, type, String.to_integer(amount)) do
+      conn
+      |> send_resp(200, "Quests added")
+    end
+  end
+
+  def complete_quest(conn, %{"user_id" => user_id, "quest_id" => quest_id}) do
```

Comment:
Why does the client need to ask us to complete a quest? Shouldn't we (backend) be doing this automatically?

Reply:
Now the client will be the one completing question by taping a "claim" button, that's why we need this endpoint